pkg/nodeps/utils.go

```diff
@@ -33,10 +37,55 @@ func IsDockerToolbox() bool {
 	return false
 }
 
+var isInternetActiveAlreadyChecked = false
+var isInternetActiveResult = false
+var isInternetActiveNetResolver interface {
+	LookupHost(ctx context.Context, host string) (addrs []string, err error)
+} = net.DefaultResolver
```

Comment:
I guess I'm not terribly happy that the desire to add a test imposes mocking complexity into the tested code here.

Reply:
Hm, do you have a better idea? Sadly there is no standard interface for the resolver (afaik) and we have to do something to replace the default one.